### PR TITLE
Docs: Updated GETDEL DOCS #779

### PR DESCRIPTION
**Pull request by surya0180**
fixes https://github.com/DiceDB/dice/issues/779

Changes
1. Created markdown tables for Parameters and Return Values
2. Updated CLI prompt to 127.0.0.1:7379>
3. Changed order of headings according to SET command

### CODE REVIEW

**Comment by surya0180** [comment]
@lucifercr07 Please review this PR. Thanks

**Comment by surya0180** [comment]
@JyotinderSingh Please review this PR. Thanks

File: docs/src/content/docs/commands/GETDEL.md [hunk 1]
```diff
@@ -1,93 +1,90 @@
...
-`Output:`
+## Examples
 
-```plaintext
+### Example with Existent key
+
+```bash
+127.0.0.1:7379> SET mykey "Hello, World!"
+OK
+127.0.0.1:7379> GETDEL mykey
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/GETDEL.md:57]: Could you please add a get command after getdel to showcase that mykey is deleted from DiceDB.

**Review by apoorvyadav1111 — CHANGES_REQUESTED** [review]
Thanks for the changes. Left minor comment.

Apoorv

File: docs/src/content/docs/commands/GETDEL.md (hunk 1 above)
Comment (surya0180) [reply docs/src/content/docs/commands/GETDEL.md:57]: Oh, how did it get missed. My Bad let me add it immediately. And thanks for reverting back @apoorvyadav1111

**Comment by surya0180** [comment]
Hey @apoorvyadav1111! I have added the GET command to demonstrate the result after the GETDEL command. Please review the changes. Thanks !

**Review by apoorvyadav1111 — APPROVED** [review]

**Pull request merged**